### PR TITLE
[Feature-WIP](inverted index) add chinese analyzer for inverted index reader

### PR DESCRIPTION
**Pull request by Tanya-W**
# Proposed changes
add chinese analyzer for inverted index reader
dependency pr: https://github.com/apache/doris/pull/14211 https://github.com/apache/doris/pull/15807 https://github.com/apache/doris/pull/15823

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/olap/rowset/segment_v2/inverted_index_reader.cpp [hunk 1]
```diff
@@ -47,29 +48,38 @@ bool InvertedIndexReader::indexExists(io::Path& index_file_path) {
...
+        InvertedIndexParserType analyser_type) {
+    std::vector<std::wstring> analyse_result;
     std::shared_ptr<lucene::analysis::Analyzer> analyzer;
+    std::unique_ptr<lucene::util::Reader> reader;
     if (analyser_type == InvertedIndexParserType::PARSER_STANDARD) {
         analyzer = std::make_shared<lucene::analysis::standard::StandardAnalyzer>();
+        reader.reset(
+                (new lucene::util::StringReader(std::wstring(value.begin(), value.end()).c_str())));
+    } else if (analyser_type == InvertedIndexParserType::PARSER_CHINESE) {
+        analyzer = std::make_shared<lucene::analysis::LanguageBasedAnalyzer>(L"chinese", false);
```
Comment (airborne12) [inline be/src/olap/rowset/segment_v2/inverted_index_reader.cpp:62]: need to set dict path for chinese analyzer here

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.1 seconds
 load time: 505 seconds
 storage size: 17122627027 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230117070049_clickbench_pr_81887.html

File: be/src/olap/rowset/segment_v2/inverted_index_reader.cpp (hunk 1 above)
Comment (Tanya-W) [reply be/src/olap/rowset/segment_v2/inverted_index_reader.cpp]: fixed

**Review by qidaye — APPROVED** [review]
LGTM

**Pull request merged**